New version: Friction.Friction version 0.9.6.1

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Related (**but failed**) #196208 

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?
  - [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

![image](https://github.com/user-attachments/assets/c9c6cdf4-72f8-4211-bba1-936d85241d19)
![image](https://github.com/user-attachments/assets/39210be5-d02e-4917-8c48-29048398348e)

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/196483)

Dragon1573 committed:
New version: Friction.Friction version 0.9.6.1

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

stephengillie commented:
URL: https://github.com/friction2d/friction/releases/download/v0.9.6.1/friction-0.9.6.1-setup-win64.exe 
Status Code: 200

(Automated message - build 900)

ItzLevvie commented:
@wingetbot run

Dragon1573 commented on manifests/f/Friction/Friction/0.9.6.1/Friction.Friction.locale.en-US.yaml:
```diff
@@ -0,0 +1,36 @@
+- animations
+- open-source
+- videos
+ReleaseNotes: "The first maintenance release for 0.9.6 is now available.This release marks the end of development for the 0.9 series. The 0.9 series is now in maintenance mode, receiving only important bug fixes until the upcoming 1.0 release later this year.See release notes for more information.\n-----BEGIN PGP SIGNED MESSAGE-----\nHash: SHA512\n\nSHA256 Checksum: \n9f90484d10a3f83eacf328173dac ...
```
```suggestion
ReleaseNotes: |-
  The first maintenance release for 0.9.6 is now available.

  This release marks the end of development for the 0.9 series. The 0.9 series is now in maintenance mode, receiving only important bug fixes until the upcoming 1.0 release later this year.

  See release notes for more information.
```

Dragon1573 commented on manifests/f/Friction/Friction/0.9.6.1/Friction.Friction.locale.en-US.yaml:
```diff
@@ -0,0 +1,36 @@
+- videos
+ReleaseNotes: "The first maintenance release for 0.9.6 is now available.This release marks the end of development for the 0.9 series. The 0.9 series is now in maintenance mode, receiving only important bug fixes until the upcoming 1.0 release later this year.See release notes for more information.\n-----BEGIN PGP SIGNED MESSAGE-----\nHash: SHA512\n\nSHA256 Checksum: \n9f90484d10a3f83eacf328173dac ...
+ReleaseNotesUrl: https://github.com/friction2d/friction/releases/tag/v0.9.6.1
+Documentations:
+- DocumentLabel: Linux | Developers
+  DocumentUrl: https://friction.graphics/documentation/source-linux.html
+- DocumentLabel: Shader Effects (Draft) | Users
+  DocumentUrl: https://friction.graphics/documentation/shaders/
+- DocumentLabel: Shortcuts (Draft) | Users
+  DocumentUrl: https://friction.graphics/documentation/shortcuts.html
+- DocumentLabel: Windows | Developers
+  DocumentUrl: https://friction.graphics/documentation/source-windows.html
```
```suggestion
```

Can't find any documentation URL from the site.

Dragon1573 committed:
Apply suggestions from code review

- Reformat "ReleaseNotes"
- Remove "Documentations" since they're no longer available

Dragon1573 commented:
> [!IMPORTANT]
>
> This package seems unable to be installed properly in _Microsoft Azure Pipelines_. Stop further attempts.

luzpaz commented:
@Dragon1573 quick question, the URLs above are from within the Friction application ? If not, where is there source from ?

Dragon1573 commented:
> The URLs above are from within the Friction application ? If not, where is there source from ?

Don't know. :disappointed: Just inherit from previous versions using _Komac_.

luzpaz commented:
@Dragon1573 those URL were removed since then. So that shouldn't be a blocker
